templates/includes/sidebar.html

```diff
@@ -0,0 +1,12 @@
+<ul class="sidebar-nav list-group">
+  {% if user.is_authenticated %}
+    <li class="list-group-item">User: <a href="{{ user.get_absolute_url }}">{{ user.get_username }}</a></li>
```

Comment:
The use of `user.get_absolute_url` assumes that the `User` model has a `get_absolute_url` method defined. Ensure that this method is implemented in the `User` model to avoid runtime errors. If not, consider removing or replacing this link.